setup.py

```diff
@@ -28,7 +28,7 @@
     ],
     keywords         = ['molecular modeling', 'computational chemistry', 'workflow', 'python interface'],
     python_requires  = '>=3.6',
-    install_requires = ['dill>=0.2.4', 'numpy', 'natsort'],
+    install_requires = ['dill>=0.2.4', 'numpy>1.23', 'scipy', 'natsort'],
```

Comment:
```suggestion
    install_requires = ['dill>=0.2.4', 'numpy', 'scipy', 'natsort'],
```
Is there a specific reason why the versions need restricting here in the first place?

Reply:
From the point of plams, I guess it ought to be agnostic to the versions. I might need to edit the build/install script instead to work around this.

Reply:
It's strange that the previous CI run was grabbing NumPy 1.23 though. I wonder if this could be related to the use of `python setup.py install` rather than `pip install .`?